met-web/src/models/subscription.ts

```diff
@@ -1,3 +1,12 @@
+export interface CreateSubscription {
```

Comment:
do we name our interrfaces as noun?
is something like SubscriptionCreationData better or SubscriptionData

Reply:
Renamed, thanks.